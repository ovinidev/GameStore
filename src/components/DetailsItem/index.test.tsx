import { render } from "@testing-library/react-native";
import { DetailsItem } from ".";

describe("DetailsItem", () => {
	it("should be able to match the snapshot", () => {
		const { toJSON } = render(<DetailsItem label="Title" value="Test" />);

		expect(toJSON()).toMatchSnapshot();
	});

	it("should be able to get text", () => {
		const { getByText } = render(<DetailsItem label="Title" value="Test" />);

		expect(getByText("Test")).toBeTruthy();
	});
});
