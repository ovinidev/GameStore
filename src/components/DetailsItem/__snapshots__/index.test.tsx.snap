// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailsItem should be able to match the snapshot 1`] = `
<View
  className="w-full"
>
  <View
    className="flex flex-row gap-2"
  >
    <Text
      className="font-semibold text-slate-100 text-xl"
    >
      Title
      :
    </Text>
    <Text
      className=" text-slate-100 text-xl"
    >
      Test
    </Text>
  </View>
</View>
`;
